src/Language/PureScript/Sugar/BindingGroups.hs

```diff
@@ -73,6 +73,7 @@ createBindingGroups moduleName = mapM f <=< handleDecls
         valueVerts = map (\d -> (d, declIdent d, usedIdents moduleName d `intersect` allIdents)) values
     bindingGroupDecls <- parU (stronglyConnComp valueVerts) (toBindingGroup moduleName)
     return $ filter isImportDecl ds ++
+             filter isExternKindDecl ds ++
```

Comment:
I'm slightly confused about this.  Without adding in this filter, does that mean that the kind declarations disappear when creating binding groups?  Also is this all that is needed here?